test/config_template.yaml

```diff
@@ -0,0 +1,55 @@
+################################################
+# Template for bioabakery snakemake workflows. #
```

Comment:
This shouldn't be here. It's a duplicate of `./config_template.yaml`. We should have a link called `config.yaml` in the test directory, so from the `./test/` directory, do `ln -f ../config_template.yaml ./config.yaml`. Then, updating the template will automatically update the thing in the test directory (and vice versa). Then delete this file (`test/config_template.yaml`)